metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -2120,6 +1271,14 @@ def get_internal_workunits(self) -> Iterable[MetadataWorkUnit]:  # noqa: C901
 
             project_name = self.get_project_name(model_name)
 
+            looker_view_id_cache: LookerViewIdCache = LookerViewIdCache(
```

Comment:
what is this used for?

Reply:
It is for the case where view is referred in SQL statement as ${view-name.SQL_TABLE_NAME}. 
Check this view [top_10_employee_income_source.view.lkml](https://github.com/sid-acryl/datahub-fork/blob/master%2Bing-510-lookml-cll/metadata-ingestion/tests/integration/lookml/vv-lineage-and-liquid-templates/top_10_employee_income_source.view.lkml). 

In this derived sql case at [L401](https://github.com/sid-acryl/datahub-fork/blob/master%2Bing-510-lookml-cll/metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py#L401) we are getting urn with `.sql_table_name` suffix and platform, database and schema as per setting available in connection, however these things are not true for derived view

At this line [L417](https://github.com/sid-acryl/datahub-fork/blob/master%2Bing-510-lookml-cll/metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py#L417), we are overwriting such urn by actual view-urn.

This cache is helping to find out LookerViewId so that we can generate the view urn and also keeping the LookerViewId in cache